infocmdb/workflow.go

```diff
@@ -13,17 +14,31 @@ import (
 // These are usually passed encoded as a json string as first process argument.
 // These are supplied to the `WorkflowFunc` of `workflow.Run`.
 type WorkflowParams struct {
-	Apikey              string `json:"apikey"`
-	TriggerType         string `json:"triggerType"`
-	WorkflowItemId      int    `json:"workflow_item_id"`
-	WorkflowInstanceId  int    `json:"workflow_instance_id"`
-	CiId                int    `json:"ciid"`
-	CiAttributeId       int    `json:"ciAttributeId"`
-	CiRelationId        int    `json:"ciRelationId"`
-	CiProjectId         int    `json:"ciProjectId"`
-	FileImportHistoryId int    `json:"fileImportHistoryId"`
+	Apikey              string 	`json:"apikey"`
+	TriggerType         string 	`json:"triggerType"`
+	WorkflowItemId      int		`json:"workflow_item_id"`
+	WorkflowInstanceId  int    	`json:"workflow_instance_id"`
+	CiId                int		`json:"ciid,string"`
+	CiAttributeId       int    	`json:"ciAttributeId"`
+	CiRelationId        int   	`json:"ciRelationId"`
+	CiProjectId         int    	`json:"ciProjectId"`
+	FileImportHistoryId int    	`json:"fileImportHistoryId"`
 }
 
+type WorkflowParamsHelper struct {
+	Apikey              string 			`json:"apikey"`
+	TriggerType         string 			`json:"triggerType"`
+	WorkflowItemId      int    			`json:"workflow_item_id"`
+	WorkflowInstanceId  int    			`json:"workflow_instance_id"`
+	CiId                IntWrapper		`json:"ciid"`
+	CiAttributeId       int    			`json:"ciAttributeId"`
+	CiRelationId        int    			`json:"ciRelationId"`
+	CiProjectId         int    			`json:"ciProjectId"`
+	FileImportHistoryId int    			`json:"fileImportHistoryId"`
+}
+
+type IntWrapper int
```

Comment:
Should be private too.
```suggestion
type intWrapper int
```